File: farmaciaAPI/DTOs/VentaDTO.cs
namespace farmaciaAPI.DTOs
{
    public class VentaDTO
    {
        public int CajaId { get; set; }
        public string TipoPago { get; set; }
        public int? ClienteId { get; set; } 
        public List<DetalleVentaDTO> Detalles { get; set; }
    }

    public class DetalleVentaDTO
    {
        public int ProductoId { get; set; }
        public int Cantidad { get; set; }
    }
}

File: POS (Point of Sale)/PharmaOneCaja/src/main/java/com/example/pharmaonecaja/dto/VentaDTO.java
package com.example.pharmaonecaja.dto;

import java.util.List;

public class VentaDTO {

    private int cajaId;
    private String tipoPago;
    private Integer clienteId;
    private List<DetalleVentaDTO> detalles;

    public int getCajaId() {
        return cajaId;
    }

    public void setCajaId(int cajaId) {
        this.cajaId = cajaId;
    }

    public String getTipoPago() {
        return tipoPago;
    }

    public void setTipoPago(String tipoPago) {
        this.tipoPago = tipoPago;
    }

    public Integer getClienteId() {
        return clienteId;
    }

    public void setClienteId(Integer clienteId) {
        this.clienteId = clienteId;
    }

    public List<DetalleVentaDTO> getDetalles() {
        return detalles;
    }

    public void setDetalles(List<DetalleVentaDTO> detalles) {
        this.detalles = detalles;
    }
}